gnosis/cowswap
# Real airdrop repo

# Summary

Fetching claim data from claim repo https://github.com/gnosis/cow-merkle-drop

![Screen Shot 2022-01-04 at 12 56 27](https://user-images.githubusercontent.com/43217/148123006-9841dedf-54d9-44e7-8294-85c088abc7fb.png)
![Screen Shot 2022-01-04 at 12 56 38](https://user-images.githubusercontent.com/43217/148123014-7662b7ed-e4f6-44d4-aa0e-327d0f2c0a3a.png)


  # To Test

1. Open the page claim page (click on `vCOW` on top)
2. Pick an address from the drop repo https://github.com/gnosis/cow-merkle-drop
3. Check it
* The account should be eligible
* The amount displayed should match what's in the repo
* Assuming no claim has been made for given account yet

**Note:** 
* Claiming not yet wired up, it won't work
* Commented out Michel's code on claim page, definitely not final

  # Background

*Optional: Give background information for changes you've made, that might be difficult to explain via comments*
